### PR TITLE
LPS-162714 | Automate LPS-159876

### PR DESCRIPTION
**Pull request by jaimelr10**
Description: Creation of Priority 4 test to verify that Friendly URL is not escaped when it has certain characters as described in [LPS-159876](https://issues.liferay.com/browse/LPS-159876)

Jira Ticket: https://issues.liferay.com/browse/LPS-162714

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/sites/friendlyurl/FriendlyURL.testcase [hunk 1]
```diff
@@ -316,4 +316,19 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-162714. Checks that Web Content FriendlyURL is not displayed escaped whit special characters like éöő"
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/sites/friendlyurl/FriendlyURL.testcase:319]: "with special characters"

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/sites/friendlyurl/FriendlyURL.testcase [hunk 2]
```diff
@@ -316,4 +316,19 @@ definition {
...
+	@priority = "4"
+	test WebContentURLNotEscaped {
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/sites/friendlyurl/FriendlyURL.testcase:321]: Please change to **SpecialCharactersAreNotEscapedFromWebContentURL**

**Review by austinchiang — CHANGES_REQUESTED** [review]
Please resend with changes, thx

**Comment by jaimelr10** [comment]
ci:test:sf

**Comment by jaimelr10** [comment]
ci:test:relevant

**Comment by jaimelr10** [comment]
ci:test:sf

**Comment by jaimelr10** [comment]
ci:test:relevant

**Comment by jaimelr10** [comment]
ci:test:relevant

**Comment by austinchiang** [comment]
ci:stop

**Comment by austinchiang** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
@jaimelr10 can you fix sf again and resend in a new pull? I'll close this one

**Pull request closed without merging**